runtime/compiler/optimizer/J9Optimizer.cpp

```diff
@@ -554,8 +556,9 @@ const OptimizationStrategy hotStrategyOpts[] =
    { OMR::earlyLocalGroup                                                 },
    { OMR::stripMiningGroup,                      OMR::IfLoops                  }, // strip mining in loops
    { OMR::loopReplicator,                        OMR::IfLoops                  }, // tail-duplication in loops
-   { OMR::blockSplitter,                         OMR::IfNews                   }, // treeSimplification + blockSplitter + VP => opportunity for EA
    { OMR::expensiveGlobalValuePropagationGroup                            },
+   { OMR::redundantGotoElimination,              OMR::IfNotJitProfiling     },
```

Comment:
Is there a reason this shouldn't be `OMR::IfEnabledAndNotJitProfiling`?

Reply:
Missed that, fixed, thanks.